### ISSUE TITLE
Missing function implementation: QVector::removeOne

### ISSUE THREAD

**Issue by de1001** [copperspice/copperspice#77]
Function is declared in the class but the implementation is missing (end result being a link error).

**Comment by agserm** [comment]
Good catch! It appears that QVector::removeOne is never called within CopperSpice. Thanks for your detailed testing, and please keep up the awesome work with your reports.
